src/controllers/authController.js

```
@@ -0,0 +1,85 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { jwtService } from '../services/jwt.service.js';
+import { tokenService } from '../services/token.service.js';
+import { userService } from '../services/user.service.js';
+import { validation } from '../utils/validation.js';
+import bcrypt from 'bcrypt';
+
+const register = async (req, res) => {
+  const { login, email, password } = req.body;
+  const errors = {
+    email: validation.validateEmail(email),
+    password: validation.validatePassword(password),
+    name: validation.validateName(login),
+  };
+
+  if (!login || !email || !password) {
+    throw ApiError.badRequest(errors);
+  }
```

This check is insufficient as it only verifies if the fields are present. It does not use the validation results from lines 11-13. You should check if the `errors` object contains any actual error messages. If it does, you need to throw an error to prevent a user with invalid data from being registered.